quarkchain/cluster/cluster_config.py

```diff
@@ -158,6 +164,7 @@ class ClusterConfig(BaseConfig):
     GRPC_SLAVE_LIST = None
     SIMPLE_NETWORK = None
     P2P = None
+    GRPC_SERVER = None
```

Comment:
can just use `GRPC_PORT` like `JSON_RPC_PORT`? then we don't need `GrpcServerConfig`